roles/sentry/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+sentry_port: 9000
```

Comment:
I added these defaults to the sentry role, as they are general variables and help ensure the role can run as expected even if we forget the variables in the inventory we pass in.